_locales/ru/messages.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extensionName": {
+        "message": "Open bookmark in Container Tab menu item",
+        "description": "Name of the extension."
+    },
+    "extensionDescription": {
+        "message": "Добавляет подменю контейнеров в контекстное меню закладок 'Открыть в контейнере'",
+        "description": "Short description of the extension."
+    },
+    "open_one_in_container_tab": {
+        "message": "Открыть в контейнере",
+        "description": "Label of item in the context menu of a bookmark. Language and capitalization should be consistent with built-in menu items such as 'Open in a New Tab'."
+    },
+    "open_all_in_container_tab": {
+        "message": "Открыть все к контейнере",
+        "description": "Label of item in the context menu of a bookmark folder. Language and capitalization should be consistent with built-in menu items such as 'Open All in Tabs'."
+    },
+    "no_container": {
+        "message": "Без контейнера",
```

Comment:
Firefox uses the following menu label. Could you update the PR to use the same, for consistency?

`Не в контейнере`

( source: https://hg.mozilla.org/l10n-central/ru/file/a247bfe885a5e449a6a93efe39c68bf258175e9c/browser/chrome/browser/browser.properties#l1084 )

Reply:
1. without container = not in container, in Russian, there is no semantic difference
2. > I'm not asking for you to add 17 characters. Rather I want the translation to fit in the UI, by using consistent language.
This will be 17 characters.
3, I'm tired of you. I just imagine how you come to the dentist with a bad tooth and start teaching him how to treat your tooth. )
Translate yourself, I see you are a pro in this.